fw-child/template/feedback.php

```diff
@@ -310,21 +428,76 @@ class="accordion-collapse collapse"
 							<div class="row row-cols-1 row-cols-md-2 row-cols-xl-3 g-4 pb-6 mb-6 border-bottom">
 								
 								<div class="col">
+
 									<label for="fullname" class="form-label"><?php _e ( 'Full Name', 'cdc' ); ?></label>
 								
 									<input type="text" name="fullname" class="form-control">
+
 								</div>
 								
 								<div class="col">
+
 									<label for="email" class="form-label"><?php _e ( 'Email Address', 'cdc' ); ?></label>
 								
 									<input type="email" name="email" class="form-control">
+
 								</div>
 								
 								<div class="col">
+
 									<label for="organization" class="form-label"><?php _e ( 'Organization', 'cdc' ); ?></label>
 								
 									<input type="text" name="organization" class="form-control">
+
+								</div>
+								
+								<div class="col">
```

Comment:
Possible to have this section span the whole row (and not be 1/3rd of the row) ?